cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowUtil.qll

```diff
@@ -484,6 +484,12 @@ predicate simpleLocalFlowStep(Node nodeFrom, Node nodeTo) {
   // Expr -> Expr
   exprToExprStep_nocfg(nodeFrom.asExpr(), nodeTo.asExpr())
   or
+  // Assignment -> LValue post-update node
```

Comment:
Let's add a comment here so we can remember why we added this rule. Suggestion:

This is used for assignments whose left-hand side is not a variable assignment or a `storeStep` but is still modeled by other means. It could be a call to `operator*` or `operator[]` where taint should flow to the post-update node of the qualifier.

Reply:
OK, done.